### PR TITLE
Thunderstorm indices

### PR DESCRIPTION
**Pull request by kieranricardo**
hey @wcarthur just making this PR for visibility on the ts indice work if you need it while I'm away.

I've run the aggregation script (but haven't checked the outputs yet) and the output is here: `/g/data/w85/QFES_SWHA/hazard/input/ts_index_climatology.nc`

### CODE REVIEW

**Comment by kieranricardo** [comment]
@wcarthur I forgot to merge this one a while back!

File: scripts/ts_indices/calculate_dowdy_ts_index.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#PBS -qnormal
+#PBS -m ae
+#PBS -M kieran.ricardo@ga.gov.au
+#PBS -l walltime=0:05:00
+#PBS -lmem=128GB,ncpus=48,jobfs=4000MB
+#PBS -joe
+#PBS -l storage=scratch/w85+gdata/w85+gdata/rt52+gdata/dk92+gdata/v10
+
+module purge
+module use /g/data/v10/public/modules/modulefiles
```
Comment (wcarthur) [inline scripts/ts_indices/calculate_dowdy_ts_index.sh:12]: Need to be sure we can use modules that are readily available. There's a bunch that are made available to all NCI users (but still need project membership). Not sure how happy DEA would be with everyone wanting to join just to use their modules.

File: scripts/ts_indices/extract_ts_indices.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
+import numpy as np
+import os
+import xarray as xr
+from calendar import monthrange
+from mpi4py import MPI
+import time
+
+references = 'Copernicus Climate Change Service (C3S) (2017): ERA5: Fifth generation of ECMWF atmospheric reanalyses of the global climate'
+
+licence = """
```
Comment (wcarthur) [inline scripts/ts_indices/extract_ts_indices.py:10]: In future we should put the C3S licence in a separate file and read it in when assigning the attribute. Makes this code a bit cleaner

**Review by wcarthur — APPROVED** [review]
Added a couple comments for consideration in the future - how to programattically include the Copernicus licence for example, rather than copying the text verbatim into our code.

**Pull request merged**